files/copr.patch

```diff
@@ -0,0 +1,4 @@
+48c48
+<     for attr in cls.__dict__:
+---
+>     for attr in list(cls.__dict__):
```

Comment:
Hard to tell without other info. Both AFAIK iterates over dict keys so I'd say there's no difference. On the other hand, I don't see any harm so I can live with this for some time.

Reply:
updated description to tell, where this came from. For context see [copr](https://pagure.io/copr/copr/blob/master/f/python/copr/v3/helpers.py#_44).

Reply:
I agree with Jirka, what is the difference? Is the reason to clone the values and don't work on "live" object?

Reply:
Yeah, I'd agree here.  Modifying (not adding/removing) should be safe with dicts.
Is this real dict, though?  Or is it some weird thing looking like dict referencing
fields in the class definition?

Reply:
Ok,  `sth.__class__.__dict__` seems to provide mappingproxy object.

Reply:
Perhaps you could use something like the following patch to see what is really
happening:

```patch
--- a/python/copr/v3/helpers.py
+++ b/python/copr/v3/helpers.py
@@ -56,6 +56,7 @@ def bind_proxy(func):
     to the actual proxy instance.
     """
     def wrapper(*args, **kwargs):
+        print("wraping " + str(func))
         result = func(*args, **kwargs)
         if type(result) not in [List, Munch]:
             return result
```

Reply:
I just realised we already have this, Sentry captures the value of *func*. It's `<function bind_proxy.<locals>.wrapper at 0x7f6e929060e0>`, though not sure what this means. Any ideas @praiskup?

Reply:
If you see "wrapper" on that place, it means that it is mapped multiple times (the bug).